dpnp/dpnp_iface_arraycreation.py

```diff
@@ -1449,22 +1447,76 @@ def meshgrid(*xi, copy=True, sparse=False, indexing="xy"):
     return call_origin(numpy.meshgrid, xi, copy, sparse, indexing)
 
 
-def mgrid_device(*, device=None, usm_type="device", sycl_queue=None):
-    return nd_grid(
-        sparse=False, device=device, usm_type=usm_type, sycl_queue=sycl_queue
-    )
+class MGridClass:
+    """
+    Construct a dense multi-dimensional "meshgrid".
+
+    For full documentation refer to :obj:`numpy.mgrid`.
+
+    Examples
```

Comment:
I believe it would be helpful to describe all parameters here (and in `OGridClass`) and to add an example with any compute follows data keyword